src/js/components/lazy-load.js

```diff
@@ -62,38 +127,54 @@ class LazyLoad {
     image.removeAttribute('data-lazy')
 
     if (placeholder.getAttribute('data-srcset')) {
-      this.parseBreakpoints(image, placeholder.getAttribute('data-srcset'))
-
-      return image
+      this.checkBreakpointsSource(image, placeholder.getAttribute('data-srcset'))
+    } else {
+      image.src = placeholder.getAttribute('data-src')
     }
 
-    image.src = placeholder.getAttribute('data-src')
-
     return image
   }
 
+  /**
+   * Parse all data-attr from [data-lazy] to <image />
+   * but prevent [data-lazy], [data-srcset] and [data-src] to be setted in img
+   *
+   * @param  {[DOM]} image
+   * @param  {[NamedNodeMap]} attributes - any html attribute from [data-lazy]
+   * @return {[DOM]} image with attributes from [data-lazy]
+   */
   parseAttributes (image, attributes) {
-    Array.prototype.forEach.call(attributes, (attr) => {
-      if (attr.name !== this.options.selector || attr.name !== 'data-srcset' || attr.name !== 'data-src') {
+    Array.from(attributes, (attr) => {
+      if (!this.isDefaultAttribute(attr.name)) {
         image.setAttribute(attr.name, attr.value)
       }
     })
 
     return image
   }
 
-  parseBreakpoints (image, breakpoints) {
-    image.removeAttribute('data-srcset')
+  /**
+   * Check if string has any of data-attr
```

Comment:
`Check if string passed as parameter matches any of the default data attributes`